products/BellHybrid/apps/common/include/common/data/BellFinishedWindowSwitchData.hpp

```diff
@@ -13,24 +13,31 @@ namespace gui
     struct BellFinishedWindowData : public gui::SwitchData
     {
       public:
+        enum class ExitBehaviour
+        {
+            SwitchWindow,
+            CloseApplication,
+            ReturnToHomescreen,
+        };
+
         struct Factory
         {
             static std::unique_ptr<BellFinishedWindowData> create(
                 const UTF8 &icon,
                 const std::string &windowToReturn,
                 const UTF8 &text                   = "",
-                bool closeApplication              = false,
+                const ExitBehaviour exitBehaviour  = ExitBehaviour::SwitchWindow,
                 const std::chrono::seconds timeout = std::chrono::seconds::zero())
             {
```

Comment:
const qualifier not nedded imo for (exitBehaviour, timeout) if parameter is not redirection(reference or pointer), but its stylistic thing

Reply:
IMHO it's fine because all the already existing fields are `const` so there is nothing to be gained. `const` makes this class unmovable, though.